__tests__/app.test.js

```diff
@@ -80,3 +80,34 @@ describe("/api/articles/:article=id", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test("GET:200 should responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeInstanceOf(Array);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("GET api/articles should return 404 for non-existent endpoint", () => {
```

Comment:
good test! Maybe should sit under its own describe block.  Not necessarily to do with endpoint api/articles!